client/src/Controller/Admin/IndexController.php

```diff
@@ -276,6 +279,10 @@ public function deleteConfirmAction($id)
 
         $this->denyAccessUnlessGranted(UserVoter::DELETE_USER, $userToDelete, 'Unable to delete this user');
 
+        if (User::ROLE_ADMIN_MANAGER === $userToDelete->getRoleName()) {
+            $this->dispatchAdminManagerDeletedEvent($userToDelete);
+        }
+
```

Comment:
Is this definitely where the user gets deleted? I can see in the `deleteAction` below this that we're calling `$this->userApi->delete()`. We would want to be sure that the user was definitely deleted so would need to wait until after this call has been made before logging an audit event.